File: doc/armstrong.java
public class Armstrong{  
  public static void main(String[] args)  {  
    int c=0,a,temp;  
    int n=153;//nombre à verifier  
    temp=n;  
    while(n>0)  
    {  
    a=n%10;  
    n=n/10;  
    c=c+(a*a*a);  
    }  
    if(temp==c)  
    System.out.println(temp+" est un nombre d'armstrong");   
    else  
        System.out.println(temp+" n'est pas un nombre d'armstrong");   
   }  
}  

File: doc/armstrong.c
#include<stdio.h>
    
int arms(nbr)
{
      int a, tmp, somme=0;  
      
      tmp=nbr;  
      
      while (nbr>0)  
      { 
        a=nbr%10;  
        somme=somme+(a*a*a);  
        nbr=nbr/10;  
      }  
      if(tmp==somme)  
        printf(" %d est un nombre Armstrong",tmp);  
      else  
        printf(" %d n'est pas un nombre Armstrong",tmp);  
      return 0;
}
int main()  
{ 
      int nbr;
      
      printf(" Entrez un nombre: ");
      scanf("%d", &nbr);
      arms(nbr);
      
      
} 
